packages/backend/src/accounting/service.ts

```diff
@@ -1,45 +1,16 @@
-import { Client } from 'tigerbeetle-node'
-import { v4 as uuid } from 'uuid'
+import { TransferError } from './errors'
 
-import {
-  AccountType,
-  calculateBalance,
-  createAccounts,
-  getAccounts
-} from './accounts'
-import {
-  areAllAccountExistsErrors,
-  BalanceTransferError,
-  CreateAccountError,
-  TransferError,
-  UnknownAccountError
-} from './errors'
-import { NewTransferOptions, createTransfers } from './transfers'
-import { BaseService } from '../shared/baseService'
-import { validateId } from '../shared/utils'
-import { toTigerbeetleId } from './utils'
-
-export enum AccountTypeCode {
-  Liquidity = 1,
-  LiquidityAsset = 2,
-  LiquidityPeer = 3,
-  LiquidityIncoming = 4,
-  LiquidityOutgoing = 5,
-  Settlement = 101
+export enum AccountType {
+  LIQUIDITY = 'LIQUIDITY',
+  LIQUIDITY_ASSET = 'LIQUIDITY_ASSET',
+  LIQUIDITY_PEER = 'LIQUIDITY_PEER',
+  LIQUIDITY_INCOMING = 'LIQUIDITY_INCOMING',
+  LIQUIDITY_OUTGOING = 'LIQUIDITY_OUTGOING',
```

Comment:
What if we have 
```
LIQUIDITY_PAYMENT
LIQUIDITY_ASSET
LIQUIDITY_PEER
LIQUIDITY_WEBMON
SETTLEMENT
```

Would this also make sense to remove `LIQUIDITY_` though?

I do like the idea of having `accountType` required in the `createLiquidityAccount` call in general, however.

Reply:
I'd lean towards `*OUTGOING` & `*INCOMING` over `*PAYMENT`

re: `LIQUIDITY_`
I just prefer `LiquidityAccountType.ASSET` over `LiquidityAccountType.LIQUIDITY_ASSET`

Reply:
makes sense. I was thinking we can keep `LIQUIDITY_ ` on the `AccountType`, because the `LiquidityAccountType` is just a subset of `AccountType` so you can't pass in `AccountType.SETTLEMENT` for `createLiquidityAccount`. 

However, I can just make a separate `AccountType` on the `LedgerAccount` Model _only_, and cast it from `LiquidityAccountType` if necessary